packages/headless/src/controllers/facet-manager/headless-facet-manager.ts

```diff
@@ -59,10 +66,17 @@ export function buildFacetManager(engine: Engine<SearchSection>): FacetManager {
     },
 
     get state() {
-      const facets = engine.state.search.response.facets;
+      const facets = getState().search.response.facets;
       const facetIds = facets.map((f) => f.facetId);
 
       return {facetIds};
     },
   };
 }
+
+function loadFacetManagerReducers(
+  engine: Engine<unknown>
+): engine is Engine<SearchSection> {
```

Comment:
is there a section missing?

Reply:
It is technically missing, but I left `facetOptions` out since we do not access it's state inside the controller, so it is not really needed atm.